flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/JdbcDialects.java

```diff
@@ -39,27 +39,27 @@
 /**
  * Default JDBC dialects.
  */
-public final class JDBCDialects {
+public class JdbcDialects {
```

Comment:
And what about moving each dialect in its own class (instead of having all the implementation insidide this class)? Or better: why don't load them via SPI (so we could support more dialects just adding external jars to Flink lib directory without requiring to have them in the official jdbc connector code)?